docs/logger-mdc-instrumentation.md

```diff
@@ -19,11 +19,11 @@ event's MDC copy:
   (same as `Span.current().getSpanContext().getTraceFlags().asHex()`).
 
 Those three pieces of information can be included in log statements produced by the logging library
-by specifying them in the pattern/format. Example for Spring Boot configuration (which uses
-logback):
+by specifying them in the pattern/format. 
 
+Tip: for Spring Boot configuration which uses logback, you can add MDC to log lines by overriding only the logging.pattern.level:
 ```properties
-logging.pattern.console = %d{yyyy-MM-dd HH:mm:ss} - %logger{36} - %msg trace_id=%X{trace_id} span_id=%X{span_id} trace_flags=%X{trace_flags} %n
+logging.pattern.level = %prefix(%mdc{trace_id}) %prefix(%mdc{span_id}) %prefix(%mdc{trace_flags}) %5p
```

Comment:
I had to look up what `%prefix` did, it looks cool, but I think maybe better to keep the example here simple to understand

Reply:
Hm, I'd say that also before the change, one would need some understanding of logback and look up what `%X` actually means. I've followed your advice though, but kept the more explicit `%mdc`.